virtualenvsetup.py

```diff
@@ -0,0 +1,77 @@
+import textwrap
+
+from buildbot import locks
+from buildbot.steps.shell import ShellCommand
+
+# only allow one VirtualenvSetup to run on a slave at a time.  This prevents
+# collisions in the shared package cache.
+veLock = locks.SlaveLock('veLock')
```

Comment:
while we follow Twisted's naming convention, I do not see why we should follow it here.